### DISCUSSION TITLE
v2.1.1

### DISCUSSION THREAD

**Discussion by osy** [utmapp/UTM#2568] [Releases]
This release brings two major features:

USB support is here for macOS *and* jailbroken iOS. You can attach a device to your VM with the new USB icon in the toolbar. You can enable USB 3.0 in the Share settings for faster devices on supported targets. Note that older versions of Windows (e.g. Windows 7 and below) require installing additional drivers for USB 3.0.

UTM SE is a new build for iOS that does not require JIT (and therefore is much slower). It can therefore be sideloaded on any iOS device that is not jailbroken. Jailbreak users are still recommended to install the standard UTM DEB package for faster JIT speeds and USB support.

## Installation
Visit [https://getutm.app/install/][1] for the most up to date installation instructions.

## Changes (v2.1.1)
* (iOS) Fix USB attach on (jailbroken) iOS 14.6
* Fixed slow UEFI boot due to accidentally including a debug build of EDK2
* (macOS) Fixed captured mouse not being released when VM is shut down or suspended
* (macOS) Trackpad scroll speed made more consistent (#2400)
* (macOS) Fixed Windows 10 not booting on M1 Macs (#2546)
* (macOS) Fixed Screamer audio (#2548) Note that you must now select the 'screamer' audio card for the 'mac99' system target for sound to work.

## Changes (v2.1.0)
* QEMU updated from v5.2.0 to v6.0.0
* USB support: attach a USB device to the VM (macOS and jailbroken iOS 14.5 and below only)
* UTM SE (Slow Edition): JITless iOS build available to sideload on *any* non-jailbroken device (thanks @ktemkin)
* Screamer audio device for PPC mac99 machines

Please check the full list on [Github][2] and help report any bug you find that is not listed.

[1]: https://getutm.app/install/
[2]: https://github.com/utmapp/UTM/issues

<hr /><em>This discussion was created from the release <a href='https://github.com/utmapp/UTM/releases/tag/v2.1.1'>v2.1.1 (Beta)</a>.</em>

**Comment by brunocastello** [comment]
Thanks osy! Just installed it right now on both my Mac and iPad Pro. We now have some improvements - Issue #2548 is fixed for macOS build, but not for iOS build. Coreaudio API calls for iOS are different maybe?

Mouse capture is indeed fixed as well, I noticed that. BTW the keyboard shortcut for mouse capture is the same as in vanilla QEMU? control + alt + g?

Once again, congrats for these great developments for what is shaping up to be what I believe the number 1 replacement for VMware/Parallels on my future M1 Mac next year.

**Reply by lokingala** [reply]
Hi

**Comment by Chick92** [comment]
Out of curiosity, how much slower is UTM SE? 
If I’d have known about UTM a few months ago I never would have upgraded my iPad!

**Reply by osy** [reply]
My own experience is like 10x slower but it differs in different OSes.

**Reply by eduo** [reply]
Does UTM SE take advantage of jit if it’s been enabled (Altjit, jitterbug) and if it hasn’t then it runs slower or it will never take advantage of JIT even when enabled for the app? (Always speaking of jailed devices)

**Reply by osy** [reply]
No it doesn’t, you need to download the regular utm

**Reply by eduo** [reply]
That's a shame. I can usually enable JIT but sometimes I can't. It would be nice if I could accept the speed hit in those times but use JIT the rest of the time, instead of having two separate apps and try to juggle having both starting the same virtual machines.
